.github/workflows/docker.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Checkout heffte
         # actions/checkout doesn't work for external repos yet
         run: |
-          git clone --depth 1 --branch v2.0.0 https://bitbucket.org/icl/heffte.git heffte
+          git clone --depth 1 --branch v2.2.0 https://bitbucket.org/icl/heffte.git heffte
```

Comment:
Was that used at all?  We were requiring v2.1.0 so I am not sure what was going on.

Reply:
I am not sure that Cabana_REQUIRE_HEFFTE is being set in the CI everywhere it should be

Reply:
You may be right, but @junghans any idea why all the Cajita runs are failing on the docker build?